go.mod

```diff
@@ -12,8 +12,8 @@ require (
 	github.com/gorilla/sessions v1.2.1
 	github.com/gosimple/slug v1.12.0
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
+	github.com/improbable-eng/grpc-web v0.15.0
 	github.com/joho/godotenv v1.4.0
-	github.com/labstack/echo-contrib v0.12.0
 	github.com/labstack/echo/v4 v4.7.2
```

Comment:
Still here; seems to be a single method `login()` on the `testStore` still depending on echo in `web/auth/auth_test.go:359`. The `login()` method is not used by any code at the moment, but not sure tests are going to need it later... 

You can use:
```sh
go mod why github.com/labstack/echo/v4
```
To find where a dependency is being used.

Reply:
Yup, tests need a big rewrite. Will tidy the module files after that.